sympy/core/numbers.py

```diff
@@ -3916,6 +3917,18 @@ def _mpc_(self):
 def _eval_is_eq(self, other): # noqa: F811
     return False
 
+@dispatch(Number, Number)
+def _eval_is_ge(lhs, rhs):
+    if lhs.is_comparable and rhs.is_comparable:
+        dif = (lhs - rhs).evalf(2)
```

Comment:
The subclasses of `Number` are `Rational`, `Float`, `Infinity`, `NegativeInfinity` and `NaN`. It shouldn't be necessary to use `.evalf` for any of these. There should probably be separate handlers for each of those classes.